app.js

```diff
@@ -1,17 +1,19 @@
 const express = require("express");
-const {all} = require("./Controllers/All.controller.js")
+const { all } = require("./Controllers/All.controller.js");
```

Comment:
This is not a good name for the controller. I can see what this is doing, but the fact you had to explain it in a comment suggests it is not named very well